Doc/library/enum.rst

```diff
@@ -739,9 +739,11 @@ Some rules:
    :meth:`__str__` and :meth:`__repr__` respectively; other codes (such as
    `%i` or `%h` for IntEnum) treat the enum member as its mixed-in type.
 5. :ref:`Formatted string literals <f-strings>`, :meth:`str.format`,
-   and :func:`format` will use the mixed-in
-   type's :meth:`__format__`.  If the :class:`Enum` class's :func:`str` or
-   :func:`repr` is desired, use the `!s` or `!r` format codes.
+   and :func:`format` will use the mixed-in type's :meth:`__format__`
+   unless :meth:`__str__` or :meth:`__format__` is overridden in the subclass,
+   in which case the overridden methods or :class:`Enum` methods will be used.
+   Use the !s and !r format codes to force usage of the :class:Enum class's
+   :func:str and :func:repr methods.
```

Comment:
Argh.  The backticks I had in my comment were interpreted as markdown.  Please add them back to \``Enum`\`, \``str`\`, and \``repr`\`.

Reply:
Oops, I should have noticed that too. Thanks for the careful review!
Also, it seems like we should make this consistent with the line above -- when we're talking about what methods are used on the class, are we talking about:


a.
```
the class's :func:`str`, :func:`format` and :func:`repr` methods
```
or b.
```
the class's :meth:`__str__`, :meth:`__format__` and :meth:`__repr__` methods
```

I'm not super familiar with this reStructuredText docs format so thanks for the guidance.

Reply:
Good catch!  In line 741 \``str.format`\` should be \``str.__format__`\`.  Still with the preceding `:meth:`

Thanks!

Reply:
Are you sure about that? `str.format` on line 741 is the main one that I wouldn't expect to change here - it's referring to something like
```
"I'm a str but I want to mention {}".format(MyEnum.foo)
```
with no `__dunders__` to be seen.

I was asking mainly about the consistency between line 743 
```
unless :meth:`__str__` or :meth:`__format__` is overridden
```
and line 745-746 
```
to force usage of the :class:`Enum` class's
:func:`str` and :func:`repr` methods.
```

Reply:
You are correct.  741 is good, 745-6 should have the dunders added to `str` and `repr`.

Reply:
OK, I also updated those to `:meth:` instead of `:func:` based on usage in the rest of the doc. I have made the requested changes; please review again